app/views/verify/sessions/new.html.slim

```diff
@@ -59,7 +59,7 @@ h1.h3.my0 = @view_model.title
       input_html: { class: 'ssn', value: @view_model.idv_form.ssn }
   .mt3
     button type='submit' class='btn btn-primary btn-wide' = t('forms.buttons.continue')
-= render 'shared/cancel', link: verify_cancel_path
+= render 'shared/cancel', link: verify_cancel_path, decorated_session: decorated_session
```

Comment:
No need to pass in the decorated_session here since it's already available to the view automatically by being defined as a `helper_method` in ApplicationController.